guided-charts/aws-hyperpod/templates/traefik/service.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: traefik
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: traefik
+    component: router
+  annotations:
+    # Use the legacy annotation (not the newer "external") which does not work in hyperpod
+    service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
+    service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: "ip"
+    service.beta.kubernetes.io/aws-load-balancer-scheme: "internet-facing"
+    # Pass TLS traffic to Traefik as TCP to allow Traefik to handle TLS termination using Let's Encrypt
```

Comment:
sanity-check: doesn't the TLS termination occurs in the NLB in this chart? does this comment need to be updated?